tracer/test/snapshots/Security.AspNetMvc5AsmRulesToggle.Integrated.enableSecurity=False._.verified.txt

```diff
@@ -0,0 +1,152 @@
+﻿[
```

Comment:
If I am not wrong, it has the same contents as as tracer/test/snapshots/Security.AspNetMvc5AsmRulesToggle.Classic.enableSecurity=False._.verified.txt 

Maybe, the same file can be used.